core/src/main/scala/org/apache/spark/ui/exec/ExecutorThreadDumpPage.scala

```diff
@@ -60,44 +61,49 @@ private[ui] class ExecutorThreadDumpPage(parent: ExecutorsTab) extends WebUIPage
           }
         }
       }.map { thread =>
-        val threadName = thread.threadName
-        val className = "accordion-heading " + {
-          if (threadName.contains("Executor task launch")) {
-            "executor-thread"
-          } else {
-            "non-executor-thread"
```

Comment:
I think it is just OK, since when the user opens the thread dump page, the task threads are always listed as the top rows of the table,